src/components/pod/pod.pw.tsx

```diff
@@ -0,0 +1,619 @@
+    [true, "1px", "solid", "rgb(204, 214, 219)"],
+    [false, "0px", "none", "rgba(0, 0, 0, 0.9)"],
+  ] as [PodProps["border"], string, string, string][]).forEach(
+    ([boolVal, px, style, color]) => {
+      test(`should render with ${boolVal} as border and ${px} as border value`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<PodExample border={boolVal} />);
+
+        const elem = podBlock(page);
+        await checkCSSOutline(elem, px, "border", style, color);
+      });
+    }
+  );
+
+  specialCharacters.forEach((children) => {
+    test(`should render with ${children} as children`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PodExample>{children}</PodExample>);
+
+      page.getByText(children);
+      await expect(podContent(page)).toHaveCSS("text-align", "left");
+    });
+  });
+
+  specialCharacters.forEach((className) => {
+    test(`should render with ${className} as className`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PodExample className={className} />);
+
+      await containsClass(podComponent(page), className);
+    });
+  });
+
+  ([
+    [SIZE.EXTRASMALL, 120, 66],
+    [SIZE.SMALL, 120, 66],
+    [SIZE.MEDIUM, 136, 82],
+    [SIZE.LARGE, 184, 98],
+    [SIZE.EXTRALARGE, 216, 130],
+  ] as [PodProps["size"], number, number][]).forEach(
+    ([size, expectedHeight, expectedWidth]) => {
+      test(`should render with ${size} as size and ${expectedHeight} as height and ${expectedWidth} as width`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<PodExample size={size} />);
+
+        await assertCssValueIsApproximately(
+          podBlock(page),
+          "height",
+          expectedHeight
+        );
+
+        await assertCssValueIsApproximately(
+          podBlock(page),
+          "width",
+          expectedWidth
+        );
+      });
+    }
+  );
+
+  ([
+    ["primary", "rgb(255, 255, 255)", "none"],
+    ["secondary", "rgb(242, 245, 246)", "none"],
+    ["tertiary", "rgb(237, 241, 242)", "none"],
+    ["tile", "rgb(255, 255, 255)", "rgba(2, 18, 36, 0.2) 0px 2px 3px 0px"],
+    ["transparent", "rgba(0, 0, 0, 0)", "none"],
+  ] as [PodProps["variant"], string, string][]).forEach(
+    ([variant, color, boxShadow]) => {
+      test(`should render with ${variant} as variant when color is ${color} and boxShadow is ${boxShadow}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<PodExample variant={variant} />);
+
+        await expect(podBlock(page)).toHaveCSS("background-color", `${color}`);
+
+        await expect(podBlock(page)).toHaveCSS("box-shadow", boxShadow);
+      });
+    }
+  );
+
+  specialCharacters.forEach((title) => {
+    test(`should render with ${title} as title`, async ({ mount, page }) => {
+      await mount(<PodExample title={title} />);
+
+      await expect(podTitle(page)).toHaveText(title);
+    });
+  });
+
+  variantTypes1.forEach((variantType) => {
```

Comment:
**comment(non-blocking)**: as `variantTypes1` and `variantTypes2` are both only used once, we could alternatively replace these variables with directing passing an array to the `forEach`:

```tsx
(["h1", "h2", "h3", "h4", "h5"] as const).forEach((variantType) => {
  // tests...
})
```